buildSrc/src/main/groovy/nullaway.jacoco-conventions.gradle

```diff
@@ -0,0 +1,46 @@
+// Mostly taken from official Gradle sample: https://docs.gradle.org/current/samples/sample_jvm_multi_project_with_code_coverage.html
+plugins {
+    id 'jacoco'
+}
+
+jacoco {
+    toolVersion = "0.8.7"
```

Comment:
Should we have this be a reference to somewhere else to keep all our tooling version numbers in the same place? Is that even desirable in this case, or will every jacoco upgrade involve a new version of this script taken from the docs?

Reply:
I don't think jacoco upgrades will necessitate a change to this script.  But, since this is in `buildSrc`, it could be kind of gross to pull in `dependencies.gradle`.  I propose leaving this one as is (I can add a comment), but generally trying to keep versions in `dependencies.gradle`.  Does that sound ok @lazaroclapp?